[36] Documentation

Drish-xD opened the pull request:
- Added basic documentation of the application
- Installation Guide
- Useful Diagrams & flows.
- Detailed E2E Guide.
- CI Guide

suryabulusu reviewed (approved):
looks great! maybe add a line about mock service worker and improve images. then we can merge. 

also can you provide a sample session json somewhere in README (or just point to e2e testing fixture) so that people get an idea of what is present in our session

Drish-xD commented:
> looks great! maybe add a line about mock service worker and improve images. then we can merge.
> 
> also can you provide a sample session json somewhere in README (or just point to e2e testing fixture) so that people get an idea of what is present in our session

Fixed the images by adding a background. 
Added a sample Sessions for both quiz and live session. 
Added a detailed guide about E2E working and CI setup.
